pupil-spa/src/app/services/check-complete/check-complete.service.spec.ts

```diff
@@ -22,6 +22,13 @@ let submissionService: SubmissionService;
 let tokenService: TokenService;
 
 describe('CheckCompleteService', () => {
+  const getItemMock = (arg, isPractice) => {
+    if (arg === 'config') {
```

Comment:
@GuyHarwood this in case the check complete service includes in the future different `storageService` `getItem` calls.
For the particular one we only want to return a fake config object if the caller's argument was `config`.